docs/source/overview.md

```diff
@@ -0,0 +1,77 @@
+# SimPhoNy: interoperability powered by ontology
+
+SimPhoNy is an ontology-based framework aimed at enabling interoperability between different simulation and data management tools, with a focus on materials science.
+
+
+
+## What can SimPhoNy be used for?
+
+### Manipulate ontology-based linked data, a format well suited for [FAIR data principles](https://en.wikipedia.org/wiki/FAIR_data).
+
+
+[Linked data](https://en.wikipedia.org/wiki/Linked_data) is a format for structured data that facilitates the interoperability among different data sources. In particular, the data is structured as a directed graph, consistent of nodes and labeled arcs. With SimPhoNy, you can not only manipulate this linked data, **but also transform existing non-linked data into linked data**.
+
+To better understand the idea of linked data, just have a quick glance at the toy example below.
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Sample linked data](./_static/img/mergedlinkeddata.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center"></figcaption>
+
+</figure>
+
+
+Moreover, in SimPhoNy, all linked data is based on [ontologies](https://en.wikipedia.org/wiki/Ontology_(information_science)), which give **meaning** to the data. Specifically, SimPhoNy works with ontologies based on the [Web Ontology Language](https://en.wikipedia.org/wiki/Web_Ontology_Language), making the data compatible with the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web).
+
+### Fetch data from a database, run a simulation and immediately store the results.
+
+Ontology-based linked data is not only well suited for the interoperability of data, but also of software tools. In SimPhoNy, one can instantiate individuals from special ontology classes called _wrappers_. These wrappers are both an ontology individual, and a software interface between the core of SimPhoNy (ontology based) and external software tools. We have already developed a few wrappers for some software tools, but you can even [develop your own](./wrapper_development.md)!
```

Comment:
You can give a link to the wrappers we have already developed _"some software tools"_. Somehow also, I don't like the phrasing "some .. tools", it's like we were developing the wrappers without purpose. We can argue here the choice of the wrappers based on our projects/needs. That surely makes a good impression on the reader.

Reply:
I reformulated the sentence and linked to our GitHub organization, as we do not have a proper list of wrappers I think. Check [58c3976](https://github.com/simphony/docs/pull/121/commits/58c39768fa9efe649198aad516cd3b56a39d9071). We may want to changed this in the future.

Reply:
Looks good to me that way. And yes, as far as I know, we do not have a list only of the wrappers